### PR TITLE
Database implementation

### PR DESCRIPTION
**Pull request by alejoreyes96**
Created a temporary structure for Flask modules. Created all database models utilizing SQLAlchemy as an Object-Relational Mapper (ORM) and made the pertinent connections to T4Loan's database at Amazon RDS.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 venv/
-.idea/
\ No newline at end of file
+.idea/
+instance/
```
Comment (ChristianPerez34) [inline .gitignore]: Add `__pycache__/`

File: models.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
+from app import db
+
+
+class Address(db.Model):
+    __tablename__ = 'Address'
+    # __table_args__ = {'extend_existing': True}
```
Comment (ChristianPerez34) [inline models.py]: Se necesita para luego?

File: models.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    role_type = db.Column(db.VARCHAR(10), nullable=False)
+
+
+class User(db.Model):
+
+    __tablename__ = 'User'
+    # __table_args__ = {'extend_existing': True}
+    uid = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    is_active = db.Column(db.Boolean, nullable=False)
+    u_password = db.Column(db.VARCHAR(16), unique=True, nullable=False)
```
Comment (ChristianPerez34) [inline models.py]: No puede haber passwords iguales para diferentes usuarios? May cause user confucion as to why he cannot register if he puts valid info but a repeated password.

Comment (alejoreyes96) [reply models.py]: No que lo que se iba a duardar en la base de datos era un token, no el password como tal?

File: models.py (hunk 2 above)
Comment (alejoreyes96) [reply models.py]: Es bueno tenerlo para luego, en caso de manejar tablas existentes

File: models.py (hunk 3 above)
Comment (alejoreyes96) [reply models.py]: En caso de ser un token debido a algun tipo de encripcion, debe ser unico

File: app.py [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
-
 from resources import Index
+from flask_sqlalchemy import SQLAlchemy
+import os
 
-app = Flask(__name__)
-api = Api(app)
 
-api.add_resource(Index, '/')
+my_instance_location = os.path.join(os.path.abspath(__file__), "../instance")
```
Comment (cfboy) [inline app.py]: @alejoreyes96  no habia que cambiar esto?

**Comment by alejoreyes96** [comment]
> How come some models have an object representation and others don't? @alejoreyes96

I left the repr method on the firts methods as per SQLAlchemy quickstart, but I wasn't entirely sure wether or not it is needed for every model or how are they used. I left it for future uses, when database testing was done.

Should I leave it, or remove it?

**Comment by ChristianPerez34** [comment]
> > How come some models have an object representation and others don't? @alejoreyes96
> 
> I left the repr method on the firts methods as per SQLAlchemy quickstart, but I wasn't entirely sure wether or not it is needed for every model or how are they used. I left it for future uses, when database testing was done.
> 
> Should I leave it, or remove it? @ChristianPerez34

Nah no worries, we can leave it for now and get back to it if we really need it

**Pull request merged**